guides/common/modules/proc_registering-clients-using-host-registration.adoc

```diff
@@ -0,0 +1,6 @@
+ifdef::orcharhino,satellite[]
+[id="Registering_Clients_Using_Host_Registration_{context}"]
+= Registering Clients Using Host Registration
+endif::[]
```

Comment:
I do not quite understand why title is in ifdefs here.

Reply:
This is because Katello only supports the "Host Registration" feature and does not require a separate heading. See `guides/common/modules/con_registering-clients-to-the-load-balancer.adoc` and `guides/doc-Configuring_Load_Balancer/master.adoc` line 67+.